### ISSUE TITLE
Task: Test out site with screenreader

### ISSUE THREAD

**Issue by katherinedm-usds** [usds/justice40-tool#852]
## Describe the task
Try screenreader with our site once latest round of updates are merged
cc @glennettec

## Acceptance Criteria

- [x] all text elements can be highlighted and read audibly by Mac voiceover utility
- [ ] all text elements can be highlighted and read audibly by Windows voiceover utility

**Comment by katherinedm-usds** [comment]
I went through the site in English and saw that all elements involving words/text can be read out audibly by voiceover tool. Did a subset in Spanish. CEQ image in footer does not have a voiceover description.
